localisationExample/src/World.h

```diff
@@ -13,7 +13,7 @@ class World
     LandMarkList getLandMarks();
     LandMark     getLandMark(int id);
 
-    void         plotWorld(Pose robotPose, PoseList particles, int i);
+    void         plotWorld(Pose robotPose, PoseList particles, Pose AverageParticle, int i);
```

Comment:
These alignments are a real bad habbit. Are you going to extra indent all lines when you add a new variable with a long typename?

Very bad for your git history